stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/attachments/StreamAttachmentFactories.kt

```diff
@@ -48,4 +49,13 @@ public object StreamAttachmentFactories {
         ImageAttachmentFactory(),
         FileAttachmentFactory(),
     )
+
+    /**
+     * Default quoted attachment factories we provide, which can transform image, file and link attachments.
```

Comment:
Is this correct? Technically, it will try to handle any kind of attachments. E.g. what if you create a custom attachment? It will try to show the content of the attachment since the factory just checks if the attachments are not empty (and with custom attachments they're not)

Reply:
Yes, the wording should be better since the factory handles all of the attachment types.

Reply:
This will be left as is since we refactored the check for the factory.